app/actions/CommonActions.scala

```diff
@@ -62,6 +62,29 @@ object CommonActions {
       case _ => _platform
     }
 
+    def playId: Option[String] = {
+      for {
+        playSessionCookie <- request.cookies.get("PLAY_SESSION")
+        value <- Option(playSessionCookie.value)
+      } yield {
+        value.take(value.indexOf("-"))
+      }
+    }
+
+    def initialSessionId: String = {playId.getOrElse("")}
```

Comment:
I think there should be a more informative default value e.g. `unknown_play_session`

Reply:
`{}` not needed around expression